snapshots/storage/metastore_test.go

```diff
@@ -229,25 +247,10 @@ func testGetInfo(ctx context.Context, t *testing.T, _ *MetaStore) {
 
 // compare snapshot.Info Updated and Created fields by checking they are
 // within a threshold of time.Now()
-var cmpSnapshotInfo = cmp.FilterPath(
-	func(path cmp.Path) bool {
-		field := path.Last().String()
-		return field == ".Created" || field == ".Updated"
-	},
-	cmp.Comparer(func(expected, actual time.Time) bool {
-		// cmp.Options must be symmetric, so swap the args
-		if actual.IsZero() {
-			actual, expected = expected, actual
-		}
-		if !expected.IsZero() {
-			return false
-		}
-		// actual value should be within a few seconds of now
-		now := time.Now()
-		delta := now.Sub(actual)
-		threshold := 10 * time.Second
-		return delta > -threshold && delta < threshold
-	}))
+var cmpSnapshotInfo = opt.Opts(
+	cmp.FilterPath(opt.PathString("Created"), opt.CmpTime(time.Second)),
+	cmp.FilterPath(opt.PathString("Updated"), opt.CmpTime(time.Second)),
```

Comment:
I guess this would be more clear if `opt.CmpTime` were renamed to `opt.CmpTimeWithinThresold()`  or something along those lines, so that it's clear the value is the maximum allowed variation between the expected and actual values.